.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and upload to PyPI
+
+on: [push, pull_request]
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, windows-2019, macos-10.15]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.3.1
+        # to supply options, put them in 'env', like:
+        env:
+         CIBW_SKIP: '*-win32 *i686'
+
+      - uses: actions/upload-artifact@v2
+        with:
+          path: ./wheelhouse/*.whl
```

Comment:
you could give the artifact a more descriptive name.
Currently it just shows up as:
<img width="513" alt="image" src="https://user-images.githubusercontent.com/3759021/156970496-3007da89-66a6-4d57-ad61-3f7b9d029855.png">

Reply:
I didn't realize we could download these! I thought it was just a staging area to hand things over between parallel jobs. Will give it a name